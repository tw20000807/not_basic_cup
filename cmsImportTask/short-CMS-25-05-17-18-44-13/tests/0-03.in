SX4sl1lxDFZr39zzEnirMRn9hy1cLOWa
3
2 1 3