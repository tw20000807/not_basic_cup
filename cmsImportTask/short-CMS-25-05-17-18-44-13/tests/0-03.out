gCf9Urk7brY9n0BAGoLGC2RnqQx0ZL06
OK
1 3 2
